src/docs/previews/tags-input/main/tailwind.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
```

Comment:
This is an old version of the tags unit example. The newer version supports edit..

Reply:
Fixed!